File: 진우/14weeks/10872. 팩토리얼/팩토리얼.py
N = int(input())  # 정수 N 입력

# 0!은 1이므로 바로 출력
if N == 0:
    print(1)
else:
    result = 1  # 팩토리얼 결과를 저장할 변수

    # 1부터 N까지 곱해서 N! 계산
    for i in range(1, N + 1):
        result *= i

    print(result)  # 결과 출력

File: 진우/14weeks/10872. 팩토리얼/팩토리얼.cc
#include <iostream>
using namespace std;

int main() {
    int N;
    cin >> N;          // 정수 N 입력

    // 0!은 1이므로 바로 출력
    if (N == 0) {
        cout << 1 << "\n";
    } 
    else {
        long long result = 1;  // 팩토리얼 결과를 저장할 변수 (초기값 설정)

        // 1부터 N까지 곱해서 N! 계산
        for (int i = 1; i <= N; i++) {
            result *= i;
        }

        cout << result << "\n"; // 결과 출력
    }

    return 0;
}
